po/fr.po

```diff
@@ -39,7 +41,7 @@ msgstr "Réaffichage non corrigé"
 
 #: extension.js:106
 msgid "Reordering folder contents"
-msgstr "Réorganisation du contenu du dossier"
+msgstr "Réorganisation du contenu des dossiers"
```

Comment:
This is about the folders on the app grid, that users can create. Since there can be more than one, I'd use the plural here.

Reply:
Oui.

Reply:
Merci ;)